cargo-pgrx/src/command/init.rs

```diff
@@ -287,8 +288,18 @@ fn untar(bytes: &[u8], pgrxdir: &Path, pg_config: &PgConfig, init: &Init) -> eyr
         pg_config.version()?,
         unpackdir.display()
     );
-    let mut tar_decoder = Archive::new(BzDecoder::new(bytes));
-    tar_decoder.unpack(&unpackdir)?;
+
+    if bytes.starts_with(b"PK\x03\x04")
+        || bytes.starts_with(b"PK\x05\x06")
+        || bytes.starts_with(b"PK\x07\x08")
```

Comment:
one question: is this magic a true byte string or is it a `uint32_t`, in true C style?

Reply:
it's copied from wikipedia directly, so this should be a true byte string?

Reply:
It's the `zip` based file format magic bits: `50 4B 03 04`, `50 4B 05 06`, and `50 4B 07 08`. The last two bytes are designators for the archive type.

Reply:
cool.